HEATDamage.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+
+namespace PhysicalDamage.Core
+{
+    class HEATDamage:Damage
```

Comment:
Definitely want to keep the name convention the same. `HeatDamage` should be how it's named otherwise `HEATDamage` implies HEAT is an acronym. Unless it is?

Reply:
HEAT stands for High-Explosive Anti-Tank